cg/models/cg_config.py

```diff
@@ -177,6 +177,8 @@ class TaxprofilerConfig(CommonAppConfig):
     hostremoval_reference: str
     databases: str
     slurm: SlurmConfig
+    tower_binary_path: str
+    tower_pipeline: str
```

Comment:
Not for this PR but we need to finish refactoring the tower and nextflow classes and the cg config file in servers so this is not duplicated.